syntax = "proto3"; //声明了protobuf的版本

package mprpc; //声明代码所在的包（对于C++来说就是namespace）

option cc_generic_services =true;

message RPCHeader
{
    bytes service_name = 1;
    bytes method_name = 2;
    uint32 args_size = 3;
}